src/test/groovy/com/featureprobe/sdk/server/ConditionSpec.groovy

```diff
@@ -98,6 +98,20 @@ class ConditionSpec extends Specification {
         !hitMiss
     }
 
+    def "[matches invalid regex] string condition match"() {
+        when:
+        condition.setObjects(["\\\\\\"])
+        condition.setPredicate(PredicateType.MATCHES_REGEX)
+        user.with("userId", "13797347245")
+        def hitMiss = condition.matchObjects(user, segments)
+        user.with("userId", "122122")
+        def hitMiss2 = condition.matchObjects(user, segments)
```

Comment:
No need to test Invalid  regex with two users.  The latter can be omitted.

Reply:
fixed